build/common-config.sh

```diff
@@ -110,7 +110,7 @@ var_set() {
 	"
 	env_key_sort[$i]="PDO_HOSTNAME"; i=$i+1; export PDO_HOSTNAME=${env_val[PDO_HOSTNAME]}
 
-    env_val[PDO_SGX_KEY_ROOT]="${PDO_SGX_KEY_ROOT:-${SCRIPTDIR}/keys/sgx_mode_${SGX_MODE,,}}"
+    env_val[PDO_SGX_KEY_ROOT]="${PDO_SGX_KEY_ROOT:-${PDO_SOURCE_ROOT}/docker/xfer/services/keys/sgx}"
```

Comment:
why would you put the key in xfer since that is exclusively a transfer directory for transient information? common-config is ONLY used for host installs.

Reply:
To consolidate the locations that we have for storing key, and so the operations we have to execute.
The enclave signing key is another interesting example -- I'll detail this below.